projects/Mallard/src/components/front/cards.tsx

```diff
@@ -18,24 +20,28 @@ const styles = StyleSheet.create({
         flexBasis: '100%',
     },
 })
+interface SmallCardProps {
+    style: StyleProp<ViewStyle>
+    article: FrontArticle
+}
+type InjectedProps = NavigationInjectedProps<Params>
 const SmallCard = withNavigation(
     ({
         style,
-        headline,
-        kicker,
-        path,
+        article,
         navigation,
-    }: NavigationInjectedProps & {
-        style: StyleProp<ViewStyle>
-        headline: string
-        kicker: string
-        path: string
-    }) => {
+    }: SmallCardProps & Partial<InjectedProps>) => {
         const { appearance } = useArticleAppearance()
+        if (navigation == null) {
```

Comment:
think that since `withNavigation` is wrapping this we don't really need this check?

Reply:
Until we have a nicer way of using hoc's that inject parameters we will need to do this. 
It's superfluous, but doesn't involve any risk of a dangerous cast.
:(

Reply:
um it was working before this PR and I'd be keen to know why it's not working now before adding boilerplate

Reply:
Because the parameters were typed to be an empty object. You're right the rnn types are difficult and we'll need to look at properly in a different pr.

Reply:
I think this is a reasonable first step on the path to more explicit types, which is what we should be aiming towards. Maybe it just needs a TODO comment or something?